fork-visualizer/.eslintrc.json

```diff
@@ -1,3 +1,31 @@
 {
-  "extends": "next/core-web-vitals"
+  "extends": [
+	"next/core-web-vitals",
+	"eslint:recommended"
+  ],
+  "settings": {
+	"import/resolver": {
+		"alias": {
+			"map": [ ["@/*", "./src/*"] ]
+		}
+	}
+  },
+  "rules": {
+    // Stylistic
+    // ---------
+	"no-multiple-empty-lines": "error",
+    "comma-dangle": [
+      "error",
```

Comment:
Is this formatting valid? And are comments even valid in json?

Reply:
Yes they are :)